### DISCUSSION TITLE
Multilingual is too important

### DISCUSSION THREAD

**Discussion by dsanso** [WordPress/gutenberg#45818] [General]
I would like to push forward the idea that multilingual functionality should be prioritized above all else, rather than being the last addon feature once "gutenberg" is complete.

It's hard to imagine that WordPress, arguably the most important website content management system of all time, wasn't built with multilingual functionalities into its core.

Community multilingual plugins have their fair share of flaws that cannot be fixed, due to the fact that its not deeply built into all aspects of WordPress.

**Why is multilingual functionality so incredibly important?**
Because practically every single small business or freelancer outside of "The United States of America" needs their website in more than one language. Only in the United States multilingual websites are a rarity only used by giant companies.

In every other developed country, natives are multilingual, and there are plenty of foreigners living in those countries that don't even know the local language. Therefore freelancers and very small family businesses will inevitably have their website in at least two languages: their local language and English.

The fact that WordPress is fundamentally monolingual is unsurprising to me. Because it was developed primarily with the United States in mind and behind the biased minds of said folk, unaware of how the rest of the world operates. The United States is a country where ~~there is only one official language~~ people only care to use English, government websites are only needed in English. And also The United States doesn't rely very much on the international market. In smaller GDP countries even smaller businesses have to make sure their website is understood by international or neighboring country folk.
On the same note, the only developed country in western culture that does not mandate by federal law for its schools to force students to learn a second language is... unsurprising the US. Even in the UK students in elementary/middle school must choose a second language. Obviously every other country has to learn English as a second language.

I hope it didn't come across as a rant.

TL;DR: The rest of the world deeply needs multilingual support, even the smallest of businesses that have no money. Only in the United States is multilingual non-essential.

**Comment by aaronjorbin** [comment]
> The United States is a country where there is only one official language

[The United States does not have an official language](https://www.usa.gov/life-in-the-us#item-36017).

**Comment by aaronjorbin** [comment]
> The United States is a country where <strike>there is only one official language</strike> people only care to use English, government websites are only needed in English.

Latin is used extensively in the USA Legal system.  Also, [the official website of the White House (which is built on WordPress) is available in Spanish](https://www.whitehouse.gov/es/).

**Reply by dsanso** [reply]
Since it seems like we are correcting each others' sentences, ignoring the fact that we understand the point made. It's my turn now.

> Latin is used extensively in the USA Legal system. Also, [the official website of the White House (which is built on WordPress) is available in Spanish](https://www.whitehouse.gov/es/).

Latin is a dead language. You obviously mean Spanish, or at the very least Spanish spoken in Latin America. [https://en.wikipedia.org/wiki/Latin_America](https://en.wikipedia.org/wiki/Latin_America)

**Reply by dsanso** [reply]
> The United States is a country where there is only one official language people only care to use English, government websites are only needed in English.

Even if you got 25 examples of government websites, it doesn't invalidate the point I was making. That in the United States multilingual websites are not used as much as in other regions of the world.

**Comment by bph** [comment]
At WordCamp US, Matt Mullenweg was asked a similar question. You can listen to the question and Matt's answers on the [recording at the timestamp 20:48](https://videopress.com/v/BJyPZJNx?at=1246)
In short: 
- Revisions, Collaboration, and workflow needs to change and 
- They need to be in place before introducing multilingual layer. 
- Data model is complicated

**Reply by dsanso** [reply]
Thank you so much! I also really liked that you linked that video recording!

**Reply by bph** [reply]
You are welcome @Dave645 - Matt, I found made a compelling argument. 😃
